### ISSUE TITLE
can you make this use yolov8 pose 

### ISSUE THREAD

**Issue by Pickymarker** [ju1ce/Mediapipe-VR-Fullbody-Tracking#33]
can you make this use yolov8 pose https://docs.ultralytics.com/tasks/pose

**Comment by ju1ce** [comment]
Sadly, not really.

Yolov8 is only a 2D pose estimator, meaning all it does is find where certain joints are on an image. There is no depth information, or how these keypoints are positioned in 3d space, so we cannot use it for FBT.

If a model existed that output a 3D root relative skeleton same as mediapipe, then it would be possible to convert the code to use it. However, I have yet to find a model that would do that both as accurately as mediapipe, and also as fast (it runs realtime on a cpu, something that even the smallest yolov8 pose cannot do). Googles data and compute power is hard to beat...

**Comment by githubawn** [comment]
I found this repository:
https://github.com/SkalskiP/sport

**Comment by Pickymarker** [comment]
Yeah sport would be cool to have work as full body track but that requires 2 cameras
